style/themes/high-contrast-dark.css

```
@@ -929,11 +929,9 @@ table.dataTable thead .sorting_desc_disabled::after {
   border-radius: 4px;
   text-align: center;
 }
-
-table.dataTable thead .sorting::after {
+table.dataTable thead .sorting:not(.sorting_asc):not(.sorting_desc)::after {
```

I didn't test your branch yet, but I have a small suggestion:

```suggestion
table.dataTable thead .sorting:not(.sorting_asc, .sorting_desc)::after {
```